modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2927,124 +2927,138 @@ definition {
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Custom Field ${nameField}",
+					fieldName = "customObjectField${nameField}",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject${nameField}74");
+			}
+
+			for (var letter : list "A,B") {
+				ObjectAdmin.addObjectLayoutViaAPI(
+					layoutName = "Layout ${letter}",
+					objectName = "CustomObject${letter}74");
+			}
 		}
 
-		for (var nameField : list "A,B") {
-			ObjectAdmin.addObjectFieldViaAPI(
-				fieldBusinessType = "Text",
-				fieldLabelName = "Custom Field ${nameField}",
-				fieldName = "customObjectField${nameField}",
-				fieldType = "String",
-				isRequired = "false",
-				objectName = "CustomObject${nameField}74");
+		task ("And given a 1:M relationship between the Object A and Object B") {
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObjectA74",
+				objectName_2 = "CustomObjectB74",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "manyToMany");
 		}
 
-		for (var letter : list "A,B") {
-			ObjectAdmin.addObjectLayoutViaAPI(
-				layoutName = "Layout ${letter}",
-				objectName = "CustomObject${letter}74");
-		}
+		task ("And given that the layout of objects is customized with a relationship tab") {
+			for (var letter : list "A,B") {
+				ObjectAdmin.openObjectAdmin();
 
-		ObjectAdmin.addObjectRelationshipViaAPI(
-			objectName_1 = "CustomObjectA74",
-			objectName_2 = "CustomObjectB74",
-			relationshipLabel = "Relationship",
-			relationshipName = "relationship",
-			relationshipType = "manyToMany");
+				ObjectPortlet.selectCustomObject(label = "Custom Object ${letter} 74");
 
-		for (var letter : list "A,B") {
-			ObjectAdmin.openObjectAdmin();
+				ObjectAdmin.goToLayoutsTab();
 
-			ObjectPortlet.selectCustomObject(label = "Custom Object ${letter} 74");
+				ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
 
-			CreateObject.selectTitleField(fieldLabel = "Custom Field ${letter}");
+				SelectFrame(locator1 = "IFrame#IFRAME");
 
-			PortletEntry.publish();
+				Click.clickAtNotVisible(locator1 = "ObjectAdmin#ENTRY_BOOLEAN_CHECKBOX");
 
-			ObjectAdmin.goToLayoutsTab();
+				ObjectAdmin.goToLayoutTabOnLayouts();
 
-			WaitForElementPresent(locator1 = "ObjectAdmin#KEBAB_MENU");
+				ObjectAdmin.addTabFieldsOnLayout(
+					blockName = "Block 1",
+					columnsNumber = 2,
+					fieldLabelName = "Custom Field ${letter}",
+					tabName = "Field Tab");
 
-			ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
+				ObjectAdmin.addTabRelationshipsOnLayout(
+					relationshipsLabelName = "Relationship",
+					tabName = "Relationship Tab");
 
-			SelectFrame(locator1 = "IFrame#IFRAME");
+				PortletEntry.save();
+			}
+		}
 
-			Click.clickAtNotVisible(locator1 = "ObjectAdmin#ENTRY_BOOLEAN_CHECKBOX");
+		task ("And given the Custom Field is defined as a title field in the custom objects") {
```

Comment:
`And given the text field set as the title field`

We want to keep our wording consistent here since we previously used text field in one of the givens.